R/check.r

```diff
@@ -121,15 +113,11 @@ check <- function(pkg = ".",
 #'   (\code{--no-manual}).
 #' @param args Additional arguments passed to \code{R CMD check}
 #' @param env_vars Environment variables set during \code{R CMD check}
-#' @param needs_build_tools If \code{TRUE}, will ensure build tools are
-#'   available when checking the package.
-#' @param check_dir the directory in which the package is checked
 #' @param quiet if \code{TRUE} suppresses output from this function.
 check_built <- function(path = NULL, cran = TRUE,
                         check_version = FALSE, force_suggests = FALSE,
                         run_dont_test = FALSE, manual = FALSE, args = NULL,
-                        env_vars = NULL, needs_build_tools = TRUE,
```

Comment:
Yeah, I like the idea of using the pkgbuild functions. That pushes all the build tools related errors to one home.